packages/bpk-component-divided-card/src/BpkDividedCard.js

```diff
@@ -0,0 +1,105 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow strict */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { cssModules } from '../../bpk-react-utils';
+import BpkCard from '../../bpk-component-card';
+
+import STYLES from './BpkDividedCard.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+export const ORIENTATION = {
+  horizontal: 'horizontal',
+  vertical: 'vertical',
+};
+
+export type Props = {
+  primaryContent: Node,
+  secondaryContent: Node,
+  isPrimaryContentPadded: ?boolean,
+  isSecondaryContentPadded: ?boolean,
```

Comment:
I would just have one prop that is `padded` like our current Card component

As there wouldn't be an instance and it would look strange to have part padded and not

Reply:
Sorry, I don't really catch it. Do you mean that `isPrimaryContentPadded` should be kept while  `isSecondaryContentPadded` should be removed?